src/main/java/com/sourcery/gymapp/backend/authentication/dto/PasswordChangeDto.java

```diff
@@ -0,0 +1,21 @@
+package com.sourcery.gymapp.backend.authentication.dto;
+
+import com.sourcery.gymapp.backend.authentication.validation.PasswordMatch;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+@PasswordMatch(confirmPasswordField = "repeatedPassword")
+public record PasswordChangeDto(
+        @NotBlank(message = "Password is mandatory")
+        @Size(min = 8, message = "Password must be at least 8 characters long")
+        String password,
+
+
+        @NotBlank(message = "Repeated password is mandatory")
+        @Size(min = 8, message = "RepeatedPassword must be at least 8 characters long")
```

Comment:
repeatedPassword is already validated by @PasswordMatch, so there's no need to validate its size separately